homeassistant/components/auth/__init__.py

```diff
@@ -694,3 +700,28 @@ def websocket_sign_path(
             },
         )
     )
+
+
+@callback
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "auth/edit_expiry_date_refresh_token",
+        vol.Required("refresh_token_id"): str,
+        vol.Required("disable_expiry_date"): bool,
+    }
+)
+@websocket_api.ws_require_user()
+def websocket_edit_expiry_date_refresh_token(
+    hass: HomeAssistant, connection: websocket_api.ActiveConnection, msg: dict[str, Any]
+) -> None:
+    """Handle a edit expiration date from refresh token request."""
```

Comment:
Same comment as above, the name of the WS command is not aligned with the functionality